### DISCUSSION TITLE
Could not import forward_backward_convergence from alchemlyb.convergence

### DISCUSSION THREAD

**Discussion by Wangmt2023** [alchemistry/alchemlyb#313] [Q&A] [answered]
Hi all, 
I'd like to check the convergence of the calculations
The calculations works well very without convergence.
But I can not import alchemlyb.convergence. 
Once I add the command  'from alchemlyb.convergence import forward_backward_convergence'
It fail with an error 'cannot import name 'forward_backward_convergence' from 'alchemlyb.convergence'

Alchemlyb were install with miniconda3 and python3.10
Any help pointing me in the right direction would be greatly appreciated

Best,

**Comment by xiki-tempula** [comment]
Hi @Wangmt2023 Thanks for the question. I wonder if I can have the version of the alchemlyb? 
```
import alchemlyb
alchemlyb.__version__
Out[10]: '2.0.1'
```
Thanks

**Reply by Wangmt2023** [reply, marked as answer]
Thank you for your reply.
I reinstalled alchemlyb with conda and python3.9
Now the issue has been fixed. 
Thanks again,

**Reply by xiki-tempula** [reply]
Great.
